index.html

```diff
@@ -1,150 +1,237 @@
-                </ul></li>
-            <li class="main-item skills">
-                <h3>Skills</h3>
-                <ul class="sub-list">
-                    <li>Codekit</li>
-                    <li>Codepen</li>
-                    <li>GitHub</li>
-                    <li>GitLab</li>
-                    <li>Terminal</li>
-                </ul></li>
-        </ul>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button type="button" class="button">
+          Get my CV
+          <img
+            src="./assets/img/ic_download.png"
+            width="32"
+            alt="Download icon"
+            class="dl-icon"
+          />
+        </button>
+      </div>
+      <ul class="main-list">
+        <li class="main-item languages">
+          <h3>Languages</h3>
+          <ul class="sub-list">
+            <li>JavaScript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="main-item frameworks">
+          <h3>Frameworks</h3>
+          <ul class="sub-list">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="main-item skills">
+          <h3>Skills</h3>
+          <ul class="sub-list">
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>GitHub</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
     </section>
-
-</body>
-
-</html>
\ No newline at end of file
+    <section id="contact">
+      <div class="text-wrapper contact-wrapper">
+        <h1>Get started</h1>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+      </div>
+      <form action="https://formspree.io/f/xbjblgky" class="contact-form">
+        <fieldset>
+          <label for="name">Your name</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            placeholder="Enter your name"
+            maxlength="30"
+            required
+          />
+        </fieldset>
+        <fieldset>
+          <label for="email">Your email</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Enter you email"
+            required
+          />
+        </fieldset>
+        <fieldset>
+          <textarea
+            name="message"
+            id="message"
+            placeholder="Enter your message"
+            maxlength="500"
+          ></textarea>
```

Comment:
Great design @jfoyarzo 🤝

- [ ] Kindly add the `required` attribute to this textarea so as we may not submit the form if this textarea is empty.

Reply:
Good job following the advice of the previous reviewer